CMakeLists.txt

```diff
@@ -70,6 +75,17 @@ if (USE_OPENMP)
   endif ()
 endif ()
 
+set(LIBCMAES_DEPS "")
+
+if (USE_TBB)
+  find_package (TBB)
+  if (TBB_FOUND)
+    add_definitions (-DUSE_TBB)

```

Comment:
usually we define HAVE_XXX

Reply:
> usually we define HAVE_XXX

I will make it HAVE_TBB..

Reply:
@jschueller you mean for the `add_definitions`, right?

Reply:
@jschueller you mean for the add_definitions, right? Or for the CMakelists if?

Reply:
In the CMakeLists, then you'll have to change it in your code.

Reply:
> In the CMakeLists, then you'll have to change it in the cxx files.

I do not get what you mean. Do you mean:

(a)  `add_definitions(-DUSE_TBB)` to be changed to `add_definitions(-DHAVE_TBB)`, or
(b) `if (USE_TBB)` to be changed to `if (HAVE_TBB)`?